teps/0040-ignore-step-errors.md

```diff
@@ -145,13 +156,304 @@ This proposal is limited to a step within a task and does not address `pipelineT
 * A [platform team](https://github.com/tektoncd/community/blob/main/user-profiles.md#1-pipeline-and-task-authors)
   wants to share a `Task` with their team which runs the following steps in a sequence:
   * Run unit tests (which may fail)
-  * Apply a transformation to the test results (e.g. converts them to a certain format such as junit)
+  * Apply a transformation to the test results (e.g., converts them to a certain format such as junit)
   * Upload the results to a central location used by all the teams
 
+## Proposal
+
+Introduce a new section `exit` as part of the
+[steps](https://github.com/tektoncd/pipeline/docs/tasks.md#defining-steps) definition along with an `image` and `script`.
+
+```
+- name: failing-step
+  image: alpine
+  exit:
+    onError: [ continue | fail]
+```
+
+The `Step` struct will have a new field `Exit`:
+
+```go
+type Step struct {
+
+    corev1.Container `json:",inline"`
+
+    // Script is the contents of an executable file to execute.
+    Script string `json:"script,omitempty"`
+
+    // ...
+
+    // define the exiting behavior of the container in this section
+    // takes an additional field `continue`
+    // set continue to afterSuccess (default) to indicate the entrypoint to exit the taskRun if the container exits with non zero exit code
+    // set continue to always to indicate the entrypoint to continue executing the rest of the steps irrespective of the container exit code
```

Comment:
nit i think this comment is out of date but nbd :D

Reply:
yup, fixed ✔️